### ISSUE TITLE
I think we should fork and improve

### ISSUE THREAD

**Issue by siebrand** [BIAndrews/ec2tagfacts#31]
I make extensive use of this module, and it's too bad that @BIAndrews is no longer maintaining it. Has anyone significantly improved on this already, and can we maybe create a forked forge module out of it? The module has been downloaded 32M times, and the latest version has been downloaded 1.1M times at https://forge.puppet.com/modules/bryana/ec2tagfacts. I guess there are enough consumers...

Wish list would be:
* Update dependencies to latest available versions
* Use IMDSv2 by default
* Not sure about #25 

I'm interested in your thoughts.

**Comment by stevenjones-edf** [comment]
@siebrand Did you fork this module/repo in the end?

**Comment by siebrand** [comment]
@stevenjones-edf No, I didn’t fork, I refactored it out and started using tags through instance metadata. I was planning on documenting that here at some point and then never got to it. There are a few gotchas in it. At the moment I’ve completely gotten rid of this module and it has simplified installation on the puppet server side.
